frontend/model/contracts/identity.js

```diff
@@ -42,7 +45,7 @@ DefineContract({
       }
     },
     'gi.contracts/identity/updateSettings': {
-      validate: object,
+      validate: objectMaybeOf,
```

Comment:
Think you misunderstood my comment... This does nothing.

Reply:
`objectMaybeOf` should be used on line 15

Reply:
quoting from other comment:

> Edit the `validate` on line 15 to use `objectMaybeOf` instead of `objectOf` (this will help with checkpointing (#354))